lib/success.js

```diff
@@ -16,7 +16,8 @@ module.exports = async (pluginConfig, context) => {
     env: { SEMANTIC_RELEASE_PACKAGE, npm_package_name }
   } = context
   const {
-    slackWebhook = process.env.SLACK_WEBHOOK,
+    slackWebhookEnVar = 'SLACK_WEBHOOK',
```

Comment:
Where is this variable used again in this file? Apart from row 20 obviously. 😊

Reply:
Yep, it is only used at line 20. It is needed for the default `slackWebhook` value 👌.
I can create the `slackWebhook` constant outside the destructuring assignement if you think it is clearer

Reply:
I think this looks nice 😊